src/components/SiteTrackingRequired/SiteTrackingRequired.test.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { render, cleanup, waitForDomChange } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import DopplerIntlProvider from '../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+import { AppServicesProvider } from '../../services/pure-di';
+import { SiteTrackingRequired, SiteTrackingNotAvailableReasons } from './SiteTrackingRequired';
+import { async } from 'q';
```

Comment:
What is this? are you sure that it is required?

Reply:
ohh maybe when add `async` in the test autoimport this. can remove